src/emulator/storage/files.ts

```diff
@@ -280,7 +291,7 @@ export class StorageLayer {
   public oneShotUpload(
     bucket: string,
     object: string,
-    contentType: string,
+    contentType: string | undefined,
```

Comment:
This arg can be removed entirely as the content type should be stored in `incomingMetadata`.

Reply:
I believe its needed for the multi-part uploads? (Though I left the multi-part uploads alone in any refactoring)

Reply:
There are no usages in this method. It's already been removed in master so this will change when you end up rebasing